github/ci/prow-deploy/files/jobs/kubevirt/virt-template/virt-template-0.2.yaml

```
@@ -0,0 +1,147 @@
+        resources:
+          requests:
+            memory: "8Gi"
+        securityContext:
+          privileged: true
+      nodeSelector:
+        type: bare-metal-external
+  - name: pull-virt-template-test
+    branches:
+    - release-0.2
+    always_run: false
+    always_run: false
```

YAMLlint (1.37.1)

[error] duplication of key "always_run" in mapping

(key-duplicates)